### PR TITLE
blog: Turing Award 50

### PR DESCRIPTION
**Pull request by bennn**
notes, quotes, stories from the ACM celebration of the Turing Award

### CODE REVIEW

File: blog/_src/posts/2017-06-24-turing-award-50.md [hunk 1]
```diff
@@ -0,0 +1,732 @@
...
+And Unix is definitely NOT an example of a "right design"; rather it's a
+ landmark of [worse is better](https://www.dreamsongs.com/WorseIsBetter.html) design.
+
+### panel: Advances in Deep Neural Networks
+
+Panelists: Judea Pearl (JP) Michael Jordan (MJ) Fei-Fei Li (FL) Stuart Russell (SR)
+  Ilya Sutskever (IS) Raquel Urtasun (RU).
+
+- _please welcome Judea Pearl and his panel_
+- SR: _I work in all areas of AI except for deep learning_
```
Comment (gasche) [inline blog/_src/posts/2017-06-24-turing-award-50.md]: Would it maybe be possible to expand the abbreviations to full names now? I find it a bit hard to read this way.

File: blog/_src/posts/2017-06-24-turing-award-50.md [hunk 2]
```diff
@@ -0,0 +1,732 @@
+    Title: Quotes and Stories from "Turing 50"
+    Date: 2017-06-24T20:00:52
+    Tags: dear diary, by Ben Greenman
+
+The ACM recently hosted [a celebration of 50 years of the A.M. Turing award](https://www.acm.org/turing-award-50).
```
Comment (bennn) [inline blog/_src/posts/2017-06-24-turing-award-50.md]: San Francisco, at the Westin St. Francis.

(I don't want to write that down, it's in the link.)

**Comment by bennn** [comment]
Formatting is much nicer now, here's a sort-of preview (the css doesn't load):
http://www.ccs.neu.edu/home/types/test/turing50.htm

**Comment by gasche** [comment]
Thanks, will read. The trick to get the CSS working in previews is to add this after `<html>`:

    <base href="http://prl.ccs.neu.edu/" />

**Comment by bennn** [comment]
`<base` doesn't fix it.

The way I made that page was by clicking "save" when the blog was running locally. I uploaded the saved `htm` file and the directory of css and js, but `ccs.neu.edu` doesn't let them load.

**Comment by gasche** [comment]
Hah. Note that if we inserted the `base` element in the rendering pipeline (either all the time, or only when doing local builds?), we could always copy the generated `.html` of a post and put it wherever.

**Pull request merged**